shopify.app.toml

```diff
@@ -1,3 +1,11 @@
 # This file stores configurations for your Shopify app.
 
 scopes = "write_products"
+
+[webhooks]
+api_version = "2024-04"
+
+  [[webhooks.subscriptions]]
+  topics = [ "app/uninstalled" ]
+  uri = "https://example.com/webhooks"
+  compliance_topics = [ "customers/data_request", "customers/redact", "shop/redact" ]
```

Comment:
Should we add these? The Ruby template had them.

Reply:
Yes, we should be all set up to process these in `app/routes/webhooks.tsx`.